services/app-api/resolvers/attributeHelper.ts

```diff
@@ -0,0 +1,25 @@
+import { Context } from "../handlers/apollo_gql"
+import { SemanticAttributes } from '@opentelemetry/semantic-conventions';
+
+export function setContextualAttributes(name: string, user: Context["user"], span: Context["span"]): void {
```

Comment:
Can you add a quick comment on this function describing what kinds of things might be the preferred use case is for this? Feels like the success and error use case is easy to assume but this one gives me pause

Reply:
good idea

Reply:
in fact, looking at this again, I think I can set these attributes just once in the middleware, and get rid of this helper

Reply:
yeah, maybe a name that makes this more clearly tied to this being info about the specific resolver being invoked would help?